.ci/atime/tests.R

```diff
@@ -210,7 +210,7 @@ test.list <- atime::atime_test_list(
 
 # Improvement discussed in and brought by https://github.com/Rdatatable/data.table/pull/4386
 "forderv improved in #4386" = atime::atime_test(
-  N = 10^seq(3, 8), # 1e9 exceeds the runner's memory (process gets killed)
+  N = 10^seq(3, 7),
```

Comment:
default above is `N = as.integer(10^seq(1, 7, by=0.25))` so `N` does not have to be specified here, so can you please delete?